src/views/List.jsx

```diff
@@ -64,13 +73,11 @@ export function List({ listPath, data, listName, userIdFromPath }) {
 					X
 				</button>
 			)}
-			<p>Check off items as you shop, your list will reset after 24hrs.</p>
-			<p>Only manually uncheck if you didn't make the purchase.</p>
+			<p className="checkbox-prompt">Checked items will reset after 24hrs.</p>
 			<>
 				<ul>
 					<div className="column-name">
-						<p>Item Name</p>
-						<p>Purchase Priority</p>
+						<p>Item Name & Priority</p>
 					</div>
```

Comment:
`suggestion` as a user it would be really easy to associate these categories if they had similar styling to the categories themselves. You could leave the `Item Name` as is, but place `Priority` underneath it in a smaller font size and with a background color (perhaps gray or something very subtle that isn't used in the other categories).

Reply:
Ok I'll talk to the team about this styling option!

Reply:
It’s fine as is too. It’s just an idea 😊